docs/scripting/callbacks/OnPlayerWeaponShot.md

```diff
@@ -1,30 +1,30 @@
 ---
 title: OnPlayerWeaponShot
-description: This callback is called when a player fires a shot from a weapon.
+description: This callback is called when a player fires a weapon.
 tags: ["player"]
 ---
 
 ## Description
 
-This callback is called when a player fires a shot from a weapon. Only bullet weapons are supported. Only passenger drive-by is supported (not driver drive-by, and not sea sparrow / hunter shots).
```

Comment:
Are driver drive-by, sea sparrow and hunter shots now supported?

Reply:
I was going to add it as a warning, but I forgot. I'll add it pronto.